### ISSUE TITLE
Feature Request:  Split's "Join delimiter" deletes after "Split delimiter"

### ISSUE THREAD

**Issue by ghsdpolley** [gchq/CyberChef#22]
To allow for quick parsing of senders from a domain in an email list, can a delete action be added to the Split Operation's "Split delimiter" where any characters trailing the "Join delimiter" are removed?  Currently, this is being performed by a bulkier Excel "Text to Columns" delimited on "@".

For example (Note: ";" used in place of '\n\r'):
----------------------------------------------------------------------------
Join delimiter - @
Split delimiter - Deletes trailing
Input - sender1@domain.com;sender2@domain.com;sender3@domain.com
Output - sender1;sender2;sender3
----------------------------------------------------------------------------

Thanks for the consideration.

**Comment by WrathZA** [comment]
Hi @ghsdpolley 

This can be accomplished by chaining the find and replace recipe onto the split recipe.

Find attached the example:

Regex @(.*) will match everything after the @ symbol

https://gchq.github.io/CyberChef/?recipe=%5B%7B%22op%22%3A%22Split%22%2C%22args%22%3A%5B%22%3B%22%2C%22Line%20feed%22%5D%7D%2C%7B%22op%22%3A%22Find%20%2F%20Replace%22%2C%22args%22%3A%5B%7B%22option%22%3A%22Regex%22%2C%22string%22%3A%22%40(.*)%22%7D%2C%22%22%2Ctrue%2Cfalse%2Cfalse%5D%7D%5D&input=c2VuZGVyMUBkb21haW4uY29tO3NlbmRlcjJAZG9tYWluLmNvbTtzZW5kZXIzQGRvbWFpbi5jb20

**Comment by ghsdpolley** [comment]
Thanks @WrathZA !

The source list has each address on a new line, so I was able to just parse through it with the RegEx addition alone.  Processed quickly and I was able to save this as a Recipe for future use.  Appreciate the fast response and help.  RegEx is the next 'language' on my list to learn...  Will close this out, but it would be cool to see a future feature allowing a default option too.  Thanks again!
